### PR TITLE
[DSCP-306] Improve PR template

### PR DESCRIPTION
**Pull request by volhovm**
I've changed the PR template. I suggest marking the issue type with a label instead. The checklist itself is minimised and is designed to be as complete as possible.

### CODE REVIEW

**Comment by volhovm** [comment]
>"I launched all related executables (if any) and they work as expected"

I agree this should be done by every PR owner, but I wouldn't add it to the checklist because it's more basic assumption, like "the bug i fixed is fixed" or "the features i added are working/tested".

Also, bumped.

**Comment by goodsoft** [comment]
`tmp/witness.key`?

**Comment by volhovm** [comment]
@goodsoft fixed.

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -4,45 +4,24 @@
...
+
+Possible related changes (conditional):
+- [ ] I added tests if required, in case they are:
+  - Covering new introduced functionality (feature).
+  - Regression tests preventing the bug from silently reappearing again (bugfix).
+- [ ] I have checked the [sample config](/../../tree/master/docs/config-full-sample.yaml) and [launch scripts](/../../tree/master/scripts/launch) and updated them if it was necessary (especially if executables or CLIs were changed).
+- [ ] I have checked READMEs and changed them accordingly if it's necessary (the main [README.md](/README.md) as well as subproject READMEs for all related executables).
+- [ ] If a public API structure or/and data serialization was changed, I made sure that these changes are reflected in the:
+  - [Swagger](/../../tree/master/specs/disciplina) API specs.
+  - Any [related](/../../tree/master/docs/api-types.md) [documentation](/../../tree/master/docs/authentication.md).
```
Comment (flyingleafe) [inline docs/pull_request_template.md]: `docs/authentication.md` isn't there anymore, please fix, LGTM otherwise.

**Pull request merged**